dkist/io/asdf/tests/test_schema_infra.py

```diff
@@ -0,0 +1,53 @@
+import pathlib
+import itertools
+
+import dkist
+from dkist.io.asdf import entry_points
+
+repodir = pathlib.Path(dkist.__file__).parent
+
+def tagname(fulltag):
+    if isinstance(fulltag, pathlib.Path):
+        return fulltag.name.split("/")[-1]
+    return fulltag.split("/")[-1]
+
+
+def test_schema_infrastructure():
```

Comment:
Amusingly, we need tests for this test.

We should run your script and then verify that this test passes, and then construct some failing examples for this test.

I'm not exactly sure what the best way of modifying the repo and the putting it back together is though.